lib/get_menu_command.rb

```diff
@@ -0,0 +1,15 @@
+class GetMenuCommand
+  attr_reader :response
+
+  def initialize(menu)
+    @menu = menu
+  end
+
+  def response?
+    response
+  end
+
+  def run()
+    @response = "The menu for this week is: #{@menu.url}"
```

Comment:
In the same vein as Felipe's comment, it seems to be easier to just return the value you need from `run()` instead of having the temporal coupling of having to call`run` then `response?` and eventually `response`.